cve_searchsploit/main.py

```diff
@@ -149,7 +158,7 @@ def recurse(locations_found, start):
 
 
 def _search_cve_aux(cve):
```

Comment:
`print (" File: " + pdir + "/exploit-database/" + file)` in this function is not a thing anymore, change to something like `URL: github_link_to_the_sploit'